packages/devtools_app/lib/src/screens/performance/panes/raster_stats/raster_stats.dart

```diff
@@ -428,3 +431,45 @@ class _LayerImageDialog extends StatelessWidget {
     );
   }
 }
+
+class _CheckerBoardBackgroundPainter extends CustomPainter {
+  final _color1 = Colors.white;
+  final _color2 = Colors.grey.shade400;
```

Comment:
we should use colors from the colorScheme so that these aren't so jarring in dark theme. 

try these: color1 - `theme.colorScheme.background`, color2 - `theme.colorScheme.outlineVariant`